sagemaker-python-sdk/pytorch_batch_inference/sagemaker_batch_inference_torchserve.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    instance_type = "ml.c5.18xlarge"
</pre><p>Please use a smaller instance size for the PR (xl, 2xl?). You can recommend using a larger instance in markdown description.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Done</p>